.travis.yml

```diff
@@ -30,27 +30,12 @@ matrix:
       - python: 3.5
 
     include:
-    - dist: trusty
+    - dist: bionic
```

Comment:
Slightly orthogonal but is there a reason to even specify this at all? Not sure why we need to deviate from whatever the default environment provides